builders/deploy/dapp/deploy.md

```diff
@@ -0,0 +1,183 @@
+3. (Optional) You can add genesis accounts and balances. If you choose to skip this step, you can use the sudo account to create accounts and transfer funds at a later time
+4. Once you have configured the template for your Appchain, you can select **Continue** to proceed to the next step of the deployment process
+
+![Create a baseline Substrate ContainerChain with the Tanssi dApp.](/images/builders/deploy/dapp/deploy/deploy-5.png)
+
+### Custom {: #custom }
+
+If you already have a Substrate runtime built and have chosen to upload your own custom specification file, there are some requirements to be aware of that are necessary to ensure the runtime can evolve into a ContainerChain on Tanssi and run properly within the Polkadot ecosystem.
+
+Your runtime must implement the following:
+
+- The Cumulus SDK, as outlined in the [Base Setup to Connect to Polkadot](/builders/build/templates/base-setup-to-polkadot){target=_blank} section of the [Templates](/builders/build/templates){target=_blank} page
+- Tanssi modules for block production, as outlined in the [Base Setup to Support the Tanssi Protocol](/builders/build/templates/#base-setup-supporting-tanssi){target=_blank} section of the [Templates](/builders/build/templates){target=_blank} page
+
+You can upload your custom raw specification file by selecting the **Custom** template and adding your JSON specification file.
+
+![Upload a custom raw specification file to the Tanssi dApp.](/images/builders/deploy/dapp/deploy/deploy-6.png)
+
+## Reserve your Appchain ID {: #reserve-appchain-id }
+
+If you haven't already done so, you'll need to reserve your Appchain ID, which will be required to register your Appchain on the relay chain and function within the Polkadot ecosystem.
+
+To reserve your Appchain ID, you'll need to submit a transaction. Please make sure to use the account you plan to launch your Appchain with when submitting the transaction.
+
+1. To initiate the transaction, click on **Reserve Appchain ID**
+2. Your wallet will pop up, and you'll need to submit the transaction
+
+![Reserve your Appchain ID via the Tanssi dApp.](/images/builders/deploy/dapp/deploy/deploy-7.png)
+
+Once the transaction has successfully gone through, your Appchain ID will be displayed on the dApp, and you'll be able to click **Continue** to proceed to the next step. You'll notice that on your Alphanet relay chain account, 20 UNIT tokens have been removed from your transferrable balance and are now reserved.
+
+![Successfully reserved your Appchain ID via the Tanssi dApp.](/images/builders/deploy/dapp/deploy/deploy-8.png)
+
+## Generate Your Appchain Files {: #generate-appchain-files }
+
+Before you can deploy your Appchain, you'll need to generate four configuration files: 
+
+1. The raw chain specification
+2. The genesis state
```

Comment:
Add some brief info on what each of the files is? (cc @augustocollado can help)

Reply:
We can add links to the chain specs articles instead:
/builders/build/substrate/customizing-chain-specs/#generating-raw-specs-file
/builders/build/substrate/customizing-chain-specs/#genesis-state

Reply:
what link for genesis wasm? can i use same link for genesis state and tanssi genesis state? @augustocollado

Reply:
so what would i do for the wasm file?

Reply:
for wasm, we can use /learn/framework/architecture/#runtime 
Tanssi genesis state....not sure, let me check!

Reply:
For tanssi, same link as genesis state